frontend/src/scenes/dashboard/DashboardCollaborators.tsx

```diff
@@ -36,73 +40,96 @@ export function DashboardCollaboration({ dashboardId }: { dashboardId: Dashboard
     const { deleteExplicitCollaborator, setExplicitCollaboratorsToBeAdded, addExplicitCollaborators } = useActions(
         dashboardCollaboratorsLogic({ dashboardId })
     )
+    const { push } = useActions(router)
+    const { openSidePanel } = useActions(sidePanelStateLogic)
+
+    const newAccessControl = useFeatureFlag('ROLE_BASED_ACCESS_CONTROL')
+
+    if (!dashboard) {
+        return null
+    }
+
+    if (newAccessControl) {
+        return (
+            <div>
+                <h3>Access control</h3>
+                <LemonBanner type="info" className="mb-4">
+                    Permissions have moved! We're rolling out our new access control system. Click below to open it.
```

Comment:
This is a nice system

Reply:
Although we _may_ need to consider still showing the old UI if there are some configurations set. Depends on whether we will go for an all-in-one migration or not

Reply:
This only renders when the flag is on so my idea was we migrate users and then turn on the flag but that might be too simplistic so I'll keep that in mind.